docs/central-entities.rst

```diff
@@ -233,6 +233,107 @@ You can see the full XLSForm `here <https://docs.google.com/spreadsheets/d/12oJZ
+
+.. note::
+   The same rules and behaviors apply when updating Entities through updates made directly to Entities in Central's management panel, or over the API. It's possible, for example, that Bob opened an Entity data edit window in Central before Alice's update is uploaded, but he didn't actually press Submit his changes until much later. The same is true: Bob is looking at version 1 and thinking he's creating version 2.
+
+Central's behavior in these cases is basic: **it will always apply any changes it receives at the time and in the order it receives them**. This means that in this case, Bob's changes will always "win" over Alice's. Any time this happens, Central will generate a :guilabel:'conflict' warning in several places in the Central management panel so Project managers can see that a problem might have occurred.
+
+Of course, it's possible that Bob's changes and Alice's changes are totally compatible. For example, it's possible Alice only updated the circumference of the tree's trunk, while Bob only reported that moss is now growing. How often and severely conflicts create actual consistency problems with Entity data will depend on the design and complexity of your data collection process and your Forms. Central attempts to help Project managers determine the severity of conflicts by separating conflict warnings into two levels of severity.
+
+**All** conflicts surfaced by Central occur because two Entity updates were generated based off the same base version of an Entity. This circumstance is called a :guilabel:`Parallel Update` in Central. In the special case that a property included in an update has also been changed in the meantime, *and* the value currently indicated in Central is different from the new updated value, a full :guilabel:`Conflict` error will be created instead, indicating that multiple possibly conflicting values were written into the same property at once.
+
+.. _central-entities-build-update:
+
+Building a Form that updates an Entity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Updating Entities isn't too different from creating new Entities, but you'll have to specify the ``entity_id`` of the specific Entity instance you want to update in the ``entities`` sheet.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id
+ 
+  trees, ${tree}
+
+In this example, a tree in the ``trees`` Entity List will be updated when Submissions based on this Form are created. The particular tree to be updated must be given in the ``entity_id`` column, which here is an expression bound to the Submission that computes to the unique identifier of that tree for each data update.
+
+Once you have this update information set up, you can add ``save_to`` rules to your Form questions as with Entity creation:
+
+.. csv-table:: survey
+  :header: type, name, label, save_to
+ 
```

Comment:
It would be helpful to have a `tree` question so this could actually represent an update when combined with the `entities` sheet defined above. It could be a `select_one_from_file tree.csv` question

Reply:
i don't follow and i'm not very good at xlsform, can you amend the pr?